federationtypes.go

```diff
@@ -134,8 +136,10 @@ func (r RespState) Check(ctx context.Context, keyRing JSONVerifier) error {
 	}
 
 	// Check if the events pass signature checks.
+	logger.Infof("Checking event signatures for %d events of room state", len(allEvents))
 	if err := VerifyEventSignatures(ctx, allEvents, keyRing); err != nil {
-		return nil
+		logger.Warnf("Event signature check failed: %s", err)
```

Comment:
TBH I'd expect the caller to handle/log errors

Reply:
fair. removed.